### ISSUE TITLE
[App]: Bug in legends while plotting groups of runs

### ISSUE THREAD

**Issue by sidhartha-roy** [wandb/wandb#6931]
### Current Behavior

When plotting groups of runs I cannot add learning rate as a legend. As you can see in the screenshots below, I set 2 different legends alpha which is a float with the value of 0.5 and I also have a learning rate as a legend with float values between 1e-5 to 12e-5. However, the learning rate never shows up as a legend. In the simple group plot, it does not show up at all and in the parallel coordinate plot, it just shows up as a - .

### Expected Behavior

Expecting any float value range to show up in the legend.

### Steps To Reproduce

1. Create a group of more than one run and log learning rate as a hyperparameter.
2. Go to Charts > Edit Panel > Legend.
3. Choose learning rate as a legend from the drop down menu.
4. Optional: Log another float parameter other than learning rate with values between 1.0 and 0.1 to compare.

### Screenshots

![image](https://github.com/wandb/wandb/assets/31117024/738d7518-3ee3-4552-bed9-de29f8f31f57)
![image](https://github.com/wandb/wandb/assets/31117024/42b6b14a-dad1-4c54-acd0-1248924c9aaa)
![image](https://github.com/wandb/wandb/assets/31117024/6a5c1658-0b08-44e2-a9f2-c4f30296925c)

### Environment

OS:

Browsers: Chrome

Version:

**Comment by sydholl** [comment]
Internal GitHub User commented: 
### Current Behavior

When plotting groups of runs I cannot add learning rate as a legend. As you can see in the screenshots below, I set 2 different legends alpha which is a float with the value of 0.5 and I also have a learning rate as a legend with float values between 1e-5 to 12e-5. However, the learning rate never shows up as a legend. In the simple group plot, it does not show up at all and in the parallel coordinate plot, it just shows up as a - .

### Expected Behavior

Expecting any float value range to show up in the legend.

### Steps To Reproduce

1. Create a group of more than one run and log learning rate as a hyperparameter.
2. Go to Charts > Edit Panel > Legend.
3. Choose learning rate as a legend from the drop down menu.
4. Optional: Log another float parameter other than learning rate with values between 1.0 and 0.1 to compare.

### Screenshots

![image](https://github.com/wandb/wandb/assets/31117024/738d7518-3ee3-4552-bed9-de29f8f31f57)
![image](https://github.com/wandb/wandb/assets/31117024/42b6b14a-dad1-4c54-acd0-1248924c9aaa)
![image](https://github.com/wandb/wandb/assets/31117024/6a5c1658-0b08-44e2-a9f2-c4f30296925c)


### Environment

OS:

Browsers: Chrome

Version:


### Additional Context

_No response_

**Comment by umakrishnaswamy** [comment]
hey @sidhartha-roy - I was able to reproduce the error you're seeing. will be sure to flag this issue and update this thread as any progress arises

**Comment by umakrishnaswamy** [comment]
hey @sidhartha-roy - after a bit more digging, I was able to circumvent this issue by accessing the learning rate through a different label (bypassing any buggy behavior). a few questions to get a better understanding of your workflow:
- could you send a screenshot of your runs table? you should be able to access this via the runs tab of your project - I would like to see what exactly is in your config for each run as well as the labels used 

apologies for any confusion, and thank you for all the information you have provided thus far

**Comment by sidhartha-roy** [comment]
This is the runs table showing the learning rate.
![image](https://github.com/wandb/wandb/assets/31117024/9c7e9942-937e-4234-930e-3f987f53a146)
Here is the config showing the learning rate.
![image](https://github.com/wandb/wandb/assets/31117024/91f497fa-6157-4566-999c-31a72c5d1378)

**Comment by umakrishnaswamy** [comment]
hey @sidhartha-roy - I noticed the lr you are trying to access via legend is `optimizer.param_groups.base_model.lr`, whereas the values displayed on your runs table are `optimizer.param_groups.noise_layer.lr`, and I cannot seem to see anything under `optimizer.param_groups.base_model.lr` based on the images sent. would it be possible to include `optimizer.param_groups.noise_layer.lr` in the legend dropdown and let me know the output of that on the chart? also would be happy to jump on a call if you'd like to troubleshoot that way

**Comment by sidhartha-roy** [comment]
![image](https://github.com/wandb/wandb/assets/31117024/e6e24bdf-ff91-4d25-bd0c-61b6c4406e70)
I see, the issue was that I was trying to plot the optimizer.param_groups.base_model.lr but that value did not exist in my config even though it showed up. When I update the parameter with optimizer.param_groups.noise_layer.lr I am able to see it in the plot as a legend. Thanks for the help. I am closing the issue.
